dnsrecon.py

```diff
@@ -873,14 +877,17 @@ def check_bindversion(res, ns_server, timeout):
     Check if the version of Bind can be queried for.
     """
     version = ""
-    request = dns.message.make_query('version.bind', 'txt', 'ch')
-    try:
-        response = res.query(request, ns_server, timeout=timeout, one_rr_per_rrset=True)
-        if (len(response.answer) > 0):
-            print_status("\t Bind Version for {0} {1}".format(ns_server, response.answer[0].items[0].strings[0]))
-            version = response.answer[0].items[0].strings[0]
-    except (dns.resolver.NXDOMAIN, dns.exception.Timeout, dns.resolver.NoAnswer, socket.error, dns.query.BadResponse):
-        return version
+
+    if not CONFIG or not CONFIG.get("disable_check_bindversion", False):
+        request = dns.message.make_query('version.bind', 'txt', 'ch')
```

Comment:
The code below the if statement here (and in similar code in the recursive function) unchanged other than to nest it below the `if` statement and replace the `return` in the `except` statement with a `pass`. The result is a single `return` statement that logic falls through to as appropriate.